### PR TITLE
Arabic api

### PR DESCRIPTION
**Pull request by jtaco**
Integrate tawasol API in case the current language is Arabic.

### CODE REVIEW

File: src/components/Board/Board.component.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ import messages from './Board.messages';
 
 import './Board.css';
 
+import BoardSpeedDial from './BoardSpeedDial';
```
Comment (martinbedouret) [inline src/components/Board/Board.component.js]: These updates are related to a different feature , remove them from here

File: src/components/Board/BoardSpeedDial/BoardSpeedDial.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.mobile-only {
```
Comment (martinbedouret) [inline src/components/Board/BoardSpeedDial/BoardSpeedDial.css]: These updates are related to a different feature , remove them from here

File: src/components/Board/BoardSpeedDial/BoardSpeedDial.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
```
Comment (martinbedouret) [inline src/components/Board/BoardSpeedDial/BoardSpeedDial.js]: These updates are related to a different feature , remove them from here

File: src/components/Board/BoardSpeedDial/BoardSpeedDial.messages.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { defineMessages } from 'react-intl';
```
Comment (martinbedouret) [inline src/components/Board/BoardSpeedDial/BoardSpeedDial.messages.js]: These updates are related to a different feature , remove them from here

File: src/constants.js [hunk 5]
```diff
@@ -4,3 +4,7 @@ const DEV_API_URL = process.env.REACT_APP_DEV_API_URL || null;
 export const ARASAAC_BASE_PATH_API = 'https://api.arasaac.org/api/';
 export const API_URL =
   DEV_API_URL || `${window.location.protocol}//api.${host}`;
+export const TAWASOL_BASE_PATH_API =
+  'http://madaportal.org/tawasol/wp-json/custom/v2/dea44ade76/';
```
Comment (martinbedouret) [inline src/constants.js]: API token ID must be taken from an environment variable , remove it from here

File: src/components/Board/SymbolSearch/SymbolSearch.component.js [hunk 6]
```diff
@@ -124,11 +127,50 @@ export class SymbolSearch extends PureComponent {
...
+    } catch (err) {
+      return [];
+    }
+  };
+
   handleSuggestionsFetchRequested = async ({ value }) => {
-    const localSuggestions = this.getSuggestions(value);
-    const srasaacSuggestions = await this.fetchSrasaacSuggestions(value);
+    const arabic = /[\u0600-\u06FF]/;
+    let localSuggestions, srasaacSuggestions;
```
Comment (martinbedouret) [inline src/components/Board/SymbolSearch/SymbolSearch.component.js]: `srasaacSuggestions` typo??

**Review by martinbedouret — CHANGES_REQUESTED** [review]

File: src/components/Board/SymbolSearch/SymbolSearch.component.js (hunk 6 above)
Comment (jtaco) [reply src/components/Board/SymbolSearch/SymbolSearch.component.js]: El problema es el uso de 'let'?

**Review by martinbedouret — APPROVED** [review]

**Pull request merged**